packages/hagrid/hagrid/cli.py

```diff
@@ -426,6 +432,16 @@ def clean(location: str) -> None:
     is_flag=True,
     help="Render Docker Files",
 )
+@click.option(
+    "--no-warnings",
```

Comment:
Q: Since, by default, we thought of no warnings, we  could re-name it to --enable-warnings